packages/app/src/Spotlights.tsx

```diff
@@ -160,7 +160,7 @@ export const HDXSpotlightProvider = ({
   const { actions } = useSpotlightActions();
 
   return (
-    <>
```

Comment:
this is not a culprit, it would crash even when this was changed from fragment to div